com.unity.netcode.gameobjects/Editor/NetworkManagerEditor.cs

```diff
@@ -145,7 +145,7 @@ private void OnEnable()
                 var networkOverrideProp = networkPrefab.FindPropertyRelative(nameof(NetworkPrefab.Override));
                 var networkOverrideInt = networkOverrideProp.enumValueIndex;
 
-                return 10 + (networkOverrideInt == 0 ? EditorGUIUtility.singleLineHeight : (EditorGUIUtility.singleLineHeight * 2) + 5);
+                return 8 + (networkOverrideInt == 0 ? EditorGUIUtility.singleLineHeight : (EditorGUIUtility.singleLineHeight * 2) + 5);
```

Comment:
since we're touching code with magic number, wouldn't it be a good idea to add named constants for these ? Would improve readability

Reply:
That actually was [from this commit ](https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/commit/27d8f50699718261856f87b874762bc217fdc171#diff-a948ca63676afde137d24210d304927b4059ac2c3caa9ed7497b86ae872af9c0) that I merged into the branch of this PR.
You could bring it up with @MFatihMAR, but I don't want to change anything that isn't within the scope of this PR's actual changes.
Make sense?

Reply:
it was just 2 pixels higher than perfect 10 top & 10 bottom padding :(